safe/impact_functions/volcanic/volcano_point_population/impact_function.py

```diff
@@ -221,9 +230,13 @@ def run(self, layers=None):
              tr('Total population %s in the exposure layer') % format_int(
                  total_population),
              tr('People need evacuation if they are within the '
-                'volcanic hazard zones.'),
-             tr('"nodata" values in the exposure layer are treated as 0 '
-                'when counting the affected or total population')])
+                'volcanic hazard zones.')])
+
+        if nan_warning:
+            table_body.extend([
+                tr(
+                    '`No data` cells where detected in the population layer. '
+                    'This may affect the result.')])
```

Comment:
I think the message like others is better (No data were carried through etc) and we can keep message deleted in line 225 above. It's better for user to know how we treat their Nan data